app/models/workarea/flow_io/imported_item.rb

```diff
@@ -97,28 +97,34 @@ def prices
         price = {
           min_quantity: 1,
           regular: {
-            price: Money.from_amount(regular[:amount].to_f, regular[:currency]),
+            price: price_from_amount(regular[:amount].to_f, regular[:currency]),
             label: regular[:label],
             base_currency: {
-              price: Money.from_amount(regular[:base][:amount].to_f, regular[:base][:currency]),
+              price: price_from_amount(regular[:base][:amount].to_f, regular[:base][:currency]),
               label: regular[:base][:label]
             }
           }
         }
 
         if sale.present?
           price[:sale] = {
-            price: Money.from_amount(sale[:amount].to_f, sale[:currency]),
+            price: price_from_amount(sale[:amount].to_f, sale[:currency]),
             label: sale[:label],
             base_currency: {
-              price: Money.from_amount(sale[:base][:amount].to_f, sale[:base][:currency]),
+              price: price_from_amount(sale[:base][:amount].to_f, sale[:base][:currency]),
               label: sale[:base][:label]
             }
           }
         end
 
         [price]
       end
+
+      def price_from_amount(amount = nil, currency = nil)
```

Comment:
we definitely shouldn't have to do this in a normal situation, but considering that messing up Gemfile order (even letting rubocop autocorrect it for you) will cause this to fail, and since SKUs are kinda hard to delete from the flow admin (not everyone has access to, maybe?), I feel like we should be defending against this unlikely but possible scenario.